db/src/main/kotlin/edu/artic/db/models/ArticExhibition.kt

```diff
@@ -44,6 +44,11 @@ data class ArticExhibition(
             return aic_start_at.toCurrentTimeZone()
         }
 
+    val thumbUrl: String?
+        get() {
+            return "$imageUrl?w=200&h=150"
```

Comment:
If `imageUrl` is nullable, should we maybe return null or an empty string if `imageUrl` is null?

Reply:
Yea. How did i miss that. Will update it. Thanks.

Reply:
Updated.